app/scripts/views/base.js

```diff
@@ -103,7 +103,11 @@ function (Cocktail, _, Backbone, Raven, $, p, AuthErrors,
       this.fxaClient = options.fxaClient;
       this._canGoBack = options.canGoBack;
 
-      Backbone.View.call(this, options);
+      try {
+        Backbone.View.call(this, options);
+      } catch (e) {
```

Comment:
This is a workable solution, but it may not go far enough. I was digging in to how views are created in router.js, and I think right now, if a view's constructor blows up, then we'll also have a blank page. [router.js->showView](https://github.com/mozilla/fxa-content-server/blob/d47d81591ddcda9e9b2b9035fc2562f50b72d120/app/scripts/router.js#L286) attempts to render a view, and if there is an error in render, [redirect to /unexpected_error](https://github.com/mozilla/fxa-content-server/blob/d47d81591ddcda9e9b2b9035fc2562f50b72d120/app/scripts/router.js#L289). Perhaps the [entire fail block](https://github.com/mozilla/fxa-content-server/blob/d47d81591ddcda9e9b2b9035fc2562f50b72d120/app/scripts/router.js#L286-L292) should be moved to [createAndShowView](https://github.com/mozilla/fxa-content-server/blob/d47d81591ddcda9e9b2b9035fc2562f50b72d120/app/scripts/router.js#L191) so that it captures errors either creating or rendering a view.

Seem reasonable?